ublox/src/ubx_packets.rs

```diff
@@ -83,13 +83,21 @@ pub trait UbxPacketCreator {
 }
 
 /// Packet not supported yet by this crate
-#[derive(Debug)]
+#[derive(Clone, Copy, Debug)]
 pub struct UbxUnknownPacketRef<'a> {
     pub payload: &'a [u8],
     pub class: u8,
     pub msg_id: u8,
 }
 
+/// Packet not supported yet by this crate
+#[derive(Clone, Debug)]
+pub struct UbxUnknownPacketOwned {
+    pub payload: Vec<u8>,
```

Comment:
Is it possible to put this behind the `alloc` flag? (for clients who haven't defined an allocator)